scripts/_resources/r_tithe.gd

```diff
@@ -0,0 +1,12 @@
+extends RMaterialStack
+
+class_name RTithe
+
+@export var favor_per_tithe_min: int = 0
```

Comment:
All of these sshould use resource stacks instead int values to couple the appropriate material w/ the value.

Reply:
I created r_favor_rewards.gd to house a material, low-, mid-, and high-tithing favor-reward amounts, and an awarded favor cap. 
![image](https://github.com/user-attachments/assets/6046ad75-37e5-40cb-b734-b540bc5ab6f6)